### PR TITLE
#1500 Improvement on QT history log

### PR DESCRIPTION
**Pull request by warrensearle**
## What's included?
Closes #1500 
Changes the way we present the history log to display date and time of each event instead of duration since last event.
This is particularly useful where the test has been taken over multiple days (e.g. main test day plus mop-up day)

## Who should test?
✅ Product owner

## How to test?
Open the QT response page for a completed test. View the History tab and check that the log at the end displays dates & times.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Should display date & time next to each event.

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_

### CODE REVIEW

**Review by mbrookeswebdev — APPROVED** [review]

File: src/views/Exercise/Tasks/QualifyingTests/QualifyingTest/Response/View.vue [hunk 1]
```diff
@@ -826,6 +827,7 @@ export default {
             {}
           );
       }
+      console.log(ordered);
```
Comment (HalcyonJAC) [inline src/views/Exercise/Tasks/QualifyingTests/QualifyingTest/Response/View.vue:830]: Got a console log here - still needed?

**Review by HalcyonJAC — APPROVED** [review]
Approving this, even with a console log statement as it won't break it, just be annoying.

**Review by joy-ade — APPROVED** [review]
Approved by product team

**Comment by joy-ade** [comment]
@warrensearle Can you look into this ticket please?

**Pull request merged**